cmd/template/framework/files/routes/fiber.go.tmpl

```diff
@@ -27,3 +36,34 @@ func (s *FiberServer) healthHandler(c *fiber.Ctx) error {
 	return c.JSON(s.db.Health())
 }
 {{end}}
+
+{{if .AdvancedOptions.websocket}}
+func (s *FiberServer) websocketHandler(con *websocket.Conn) {
+	ctx, cancel := context.WithCancel(context.Background())
+
+	go func() {
+		for {
+			_, _, err := con.ReadMessage()
+			if err != nil {
+				cancel()
+				log.Println("Receiver Closing", err)
+				break
+			}
+		}
+	}()
+
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		default:
+			payload := fmt.Sprintf("server timestamp: %d", time.Now().UnixNano())
+			if err := con.WriteMessage(websocket.TextMessage, []byte(payload)); err != nil {
+				log.Printf("could not write to socket: %v", e)
```

Comment:
```suggestion
				log.Printf("could not write to socket: %v", err)
```

Reply:
I don't know how I didn't catch this but I also think ci should have caught a non building option.
I created [an issue](https://github.com/Melkeydev/go-blueprint/issues/195) about that which I plan to also fix.